ownership-chain/runtime/klaos/src/tests/mod.rs

```diff
@@ -3,10 +3,11 @@
 #![cfg(test)]
 mod precompile_tests;
 mod version_tests;
-mod xcm_mock;
-mod xcm_tests;
+// TODO: uncomment this once we are using deps from crates.io
```

Comment:
`orml` crates use deps from `crates.io` which causes version mismatch between `polkadot-sdk` deps and causes `xcm-tests` to fail. Generic refactor of xcm tests is needed:
1. remove dependency on `orml` crates
2. replace usage of `DmpQueue` with `MessageQueue`, see above why